docs/blog/announcing-vite3.md

```diff
@@ -62,9 +60,9 @@ The theme is now shared by all templates. This should help better convey the sco
 
 Apart from the CLI’s aesthetics improvements, you’ll notice that the default dev server port is now 5173 and the preview server listening at 4173. This change ensures Vite will avoid collisions with other tools.
 
-### Improved WebSocket Connection Strategy
+### Improved WebSocket connection strategy
```

Comment:
I think this should be uppercase just as in the docs though 🤔

Reply:
mmm... yes, @antfu, should we normalize all the titles to uppercase?

Reply:
I personally prefer to capitalize for all titles, but changing this to make it consistent with the rest of the titles. I am fine with either way as long as it's consistent :P

Reply:
Ok, changing it

Reply:
Merging this PR, and we can keep working on the original one so there are less conflicts